production-marathon/README.md

```diff
@@ -0,0 +1,58 @@
+# Production Marathon Deployment
+
+The Production Marathon deployment supports the deployment of the CAF Job Service on Mesos/Marathon. This folder contains the marathon environment and template files that are required to deploy the service application.
```

Comment:
Drop CAF from CAF Job Service. Likewise throughout remainder of document as I think it is referred to as just the Job Service in other documentation. You may also wish to mention that both the job service and job tracking worker is deployed.